### DISCUSSION TITLE
Update roadmap?

### DISCUSSION THREAD

**Discussion by jbergstroem** [saleor/saleor#9264] [General]
First of all: congrats on getting both 3.0 and 3.1 out of the door in such a swift manner!

Seeing how both now are out, it would be great to see [the roadmap](https://github.com/saleor/saleor/projects/12) being updated since I know that most of your planning work is done in a private tracker. It would help align where to plan developer efforts around the saleor ecosystem!  Thanks.

**Comment by karolkielecki** [comment]
@jbergstroem Thanks for your kind words and suggestion. Yep, we'll update our roadmap when we can. I'll keep you posted.
